### ISSUE TITLE
how to perform inference as current app.py and given inference code is not compatible?

### ISSUE THREAD

**Issue by Arslan-Mehmood1** [zhuang2002/PowerPaint#155]
def parse_args():
    args = argparse.ArgumentParser()
    args.add_argument("--pretrained_model_path", type=str, required=True)
    args.add_argument("--base_model_path", type=str, default=None)
    args.add_argument("--weight_dtype", type=str, default="float16")
    args.add_argument("--share", action="store_true")
    args.add_argument(
        "--local_files_only", action="store_true", help="enable it to use cached files without requesting from the hub"
    )
    args.add_argument("--port", type=int, default=7860)
    args = args.parse_args()

    if os.path.exists(os.path.join(args.pretrained_model_path, "brushnet")):
        args.version = "ppt2"
    else:
        args.version = "ppt1"

    if args.base_model_path is None:
        args.base_model_path = "runwayml/stable-diffusion-v1-5"
    return args

V2
# Clone PowerPaint Model
git lfs clone https://huggingface.co/JunhaoZhuang/PowerPaint_v2/ ./checkpoints/ppt-v2

python app.py --share --version ppt-v2 --checkpoint_dir checkpoints/ppt-v2

Error in loading models.

**Comment by Arslan-Mehmood1** [comment]
!git clone https://huggingface.co/JunhaoZhuang/PowerPaint-v2-1/ checkpoints/ppt-v21


ppt-v21
├── PowerPaint_Brushnet
│   ├── diffusion_pytorch_model.safetensors
│   └── pytorch_model.bin
├── realisticVisionV60B1_v51VAE
│   ├── feature_extractor
│   │   └── preprocessor_config.json
│   ├── safety_checker
│   │   ├── config.json
│   │   └── pytorch_model.bin
│   ├── scheduler
│   │   └── scheduler_config.json
│   ├── text_encoder
│   │   ├── config.json
│   │   └── pytorch_model.bin
│   ├── tokenizer
│   │   ├── merges.txt
│   │   ├── special_tokens_map.json
│   │   ├── tokenizer_config.json
│   │   └── vocab.json
│   ├── unet
│   │   ├── config.json
│   │   └── diffusion_pytorch_model.bin
│   ├── vae
│   │   ├── config.json
│   │   └── diffusion_pytorch_model.bin
│   └── model_index.json
├── .gitattributes
└── README.md
